packages/front-end/pages/power-calculator/index.tsx

```diff
@@ -126,14 +128,15 @@ const PowerCalculationPage = (): React.ReactElement => {
             setSettingsModalParams(p);
             setPowerCalculationParams(p);
             setStatsEngineSettings(modalStatsEngineSettings);
-            setShowModal(false);
+            setShowModal(null);
           }}
           statsEngineSettings={modalStatsEngineSettings}
           params={settingsModalParams}
+          startPage={showModal ?? "select"}
```

Comment:
issue: should this just be `showModal`?

This is inside a `{showModal &&` check so this is always a `PowerModalPages` already.